src/components/utils/getUniqId.ts

```diff
@@ -0,0 +1,6 @@
+import {NAMESPACE} from './cn';
+
+let id = 1;
+export function getUniqId() {
```

Comment:
Maybe we should rework [this](https://github.com/gravity-ui/uikit/blob/4d5494503c44a3b34338a072fe2a7da9d32248c6/src/components/utils/common.ts#L8) function and use it here?

Reply:
https://github.com/gravity-ui/uikit/pull/1070 – the same function will be exported here from uikit

Reply:
UPD: decided to use `tooltipId` from props